### ISSUE TITLE
Foreword is actually a prologue

### ISSUE THREAD

**Issue by robinwhittleton** [ncwalz/edgar-wallace_terror-keep#2]
Although it’s titled Foreword, this really is a prologue (“[An introductory section that sets the background to a story, typically part of the narrative](https://www.daisy.org/z3998/2012/vocab/structure/#prologue)”). So I’d change the filename, `id` and `epub:type` to `prologue`, and `frontmatter` to `bodymatter` as it’s a part of the story itself. You can then amend the manifest and spine, and regenerate the ToC.

**Comment by ncwalz** [comment]
Makes sense; thanks. Since I changed the title, I thought I should mark it as an editorial change
